docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -257,6 +441,31 @@ Compares the _context_ to specified value to determine `true` or `false`.
 Usually used in combination with <<if_fn>> or <<case_fn>>. This only works with primitive types, 
 such as `number`, `string`, and `boolean`. See also <<eq_fn>>, <<gt_fn>>, <<gte_fn>>, <<lt_fn>>, <<lte_fn>>, and <<neq_fn>>.
 
+*Expression syntax*
+[source,js]
+----
+compare “neq” to=”elasticsearch”
+compare op=”lte” to=100
+----
+
+*Code example*
+[source,text]
+----
+filters
+| demodata
+| mapColumn project 
+  fn=${getCell project | 
+    switch 
+      {case if={compare eq to=kibana} then=kibana}
+      {case if={compare eq to=elasticsearch} then=elasticsearch}
+      default="other"
+  }
+| pointseries size="size(cost)" color="project"
+| pie 
+| render
+----
+This maps all `project` values that aren’t `“kibana”` and `“elasticsearch”` to `“other”`. Alternatively, you can use the individual comparator functions instead of compare. See `eq`, `neq`, `lt`, `lte`, `gt`, and `gte`.
```

Comment:
It might be useful to link to the functions mentioned
```suggestion
This maps all `project` values that aren’t `“kibana”` and `“elasticsearch”` to `“other”`. Alternatively, you can use the individual comparator functions instead of compare. See <<eq_fn>>, <<neq_fn>>, <<lt_fn>>, <<lte_fn>>, <<gt_fn>>, and <<gte_fn>>.
```